opencti-platform/opencti-graphql/src/modules/catalog/catalog-domain.ts

```diff
@@ -159,6 +158,72 @@ const encryptValue = (publicKey: string, value: string) => {
   return encrypted.toString('base64');
 };
 
+const validateContractConfigurations = (
+  contractConfigurations: any[],
+  targetContract: CatalogContract
+) => {
+  const targetConfig = targetContract.config_schema;
+
+  // Build validation object with parsed values
+  const contractObject: any = {};
+
+  contractConfigurations.forEach((config) => {
+    const propSchema = targetConfig.properties[config.key];
+
+    if (propSchema && config.value !== undefined && config.value !== null) {
+      // Skip validation for encrypted passwords
+      if (config.encrypted) {
+        contractObject[config.key] = 'encrypted_placeholder';
```

Comment:
this function aims to validate the contract from value provided, this replacement doesnt make sens for me